node/src/transaction.test.ts

```diff
@@ -94,6 +94,23 @@ describe('Transaction', () => {
         expect(actual).toBe(expectedResult);
     });
 
+    it('sets endorsing orgs', async () => {
+        let actualOrgs: string[] | null | undefined = [];
+        client.endorse = jest.fn( req => {
+            actualOrgs = req.endorsing_organizations;
+            return Promise.resolve({
+                prepared_transaction: {
+                    payload: Buffer.from('PAYLOAD'),
+                },
+                result: {
+                    payload: Buffer.from(expectedResult),
+                },
+            });
+        })
```

Comment:
`client.endorse` is already a correctly configured mock function so you don't need to redefine it. After the call you can grab the request argument the mock captures with
```
const actualOrgs = client.endorse.mock.calls[0][0].endorsing_organizations;
```

Reply:
Thanks - that was the magic I was looking for :)